modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -27,6 +29,8 @@
     public TypeScriptNodeClientCodegen() {
         super();
 
+        typeMapping.put("file", "Buffer");
```

Comment:
Your idea is partially right but `Buffer` comes from 
[Node.js](https://nodejs.org/api/buffer.html#buffer_class_buffer). So I think I should make the diff below:

```diff
--- a/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java
+++ b/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java
@@ -29,6 +29,8 @@ public class TypeScriptNodeClientCodegen extends AbstractTypeScriptClientCodegen
     public TypeScriptNodeClientCodegen() {
         super();

+        languageSpecificPrimitives.add("Buffer");
+
         typeMapping.put("file", "Buffer");

         // clear import mapping (from default generator) as TS does not use it
```